### ISSUE TITLE
Question about lie_attack

### ISSUE THREAD

**Issue by yangjiazhne** [WenkeHuang/MarsFL#2]
### I think this is a great piece of work, but I have some doubts about certain parts of the code.
In this code, the gradient of the benign users relative to the global model is first calculated, and then the `avg` and `std `of the computed gradient are obtained. However, in the end, the gradient `avg + z * std` is directly assigned to the malicious user's **model parameters**. Shouldn't it be the **avg and std of the parameters** that are assigned instead?
```
if cfg['attack'].byzantine.evils == 'lie_attack':

        n = len(fed_method.online_clients_list)
        m = n - sum(fed_method.client_type)
        s = n // 2 + 1 - m
        z = np.exp(-1 * (s ** 2)) / (2 / ((2 * np.pi) ** 0.5))

        all_net_delta = []
        with torch.no_grad():
            for i in fed_method.online_clients_list:
                if fed_method.client_type[i] == True:
                    sele_net = fed_method.nets_list[i]

                    net_delta = []
                    for name, param0 in temp_net.state_dict().items():
                        param1 = sele_net.state_dict()[name]
                        delta = (param1.detach() - param0.detach())
                        net_delta.append(copy.deepcopy(delta.view(-1)))
                    net_delta = torch.cat(net_delta, dim=0).view(1, -1)

                    all_net_delta.append(net_delta)
            all_net_delta = torch.cat(all_net_delta, dim=0)
            avg = torch.mean(all_net_delta, dim=0)
            std = torch.std(all_net_delta, dim=0)
        for i in fed_method.online_clients_list:
            if fed_method.client_type[i] == False:
                sele_net = fed_method.nets_list[i]
                row_into_parameters((avg + z * std).cpu().numpy(), sele_net.parameters())
```

Looking forward to your explanation!
